pandas/core/accessor.py

```diff
@@ -5,15 +5,17 @@
 that can be mixed into or pinned onto other pandas classes.
 
 """
+from typing import FrozenSet, Set
 import warnings
 
 from pandas.util._decorators import Appender
 
 
 class DirNamesMixin:
-    _accessors = frozenset()
+    _accessors = set()  # type: Set[str]
```

Comment:
why did you change this?

Reply:
In `generic.py` class `NDFrame` is grandchild/child of the `DirNamesMixin`. And in class `NDFrame` and `DataFrame`, `_accessors` is a `set`, so mypy will throw an error if I leave it to be a `frozenset` here.See this PR #25909 . More specifically start reading from here https://github.com/pandas-dev/pandas/pull/25909#issuecomment-477592965

Reply:
ok that's fine